src/abv/inventory_api/style_cache.py

```diff
@@ -0,0 +1,30 @@
+

```

Comment:
All of the docstrings in this file are redundant, probably just remove them all.

Reply:
Agreed.